### PR TITLE
Update Prettier 2.0.5 -> 2.1.1

### PR DESCRIPTION
**Pull request by qmonmert**
Update Prettier 2.0.5 -> 2.1.1 for Angular, React & Vue

---

Please make sure the below checklist is followed for Pull Requests.

-   [ ] [All continuous integration tests](https://github.com/jhipster/generator-jhipster/actions) are green
-   [x] Tests are added where necessary
-   [x] Documentation is added/updated where necessary
-   [x] Coding Rules & Commit Guidelines as per our [CONTRIBUTING.md document](https://github.com/jhipster/generator-jhipster/blob/master/CONTRIBUTING.md) are followed

### CODE REVIEW

File: generators/client/templates/vue/package.json.ejs [hunk 1]
```diff
@@ -110,7 +110,7 @@ limitations under the License.
     "postcss-import": "12.0.1",
     "postcss-loader": "3.0.0",
     "postcss-url": "8.0.0",
-    "prettier": "2.0.5",
+    "prettier": "2.1.1",
```
Comment (mshima) [inline generators/client/templates/vue/package.json.ejs:113]: We should use the constant like angular.
```suggestion
    "prettier": "<%= PRETTIER_VERSION %>",
```

Comment (mshima) [reply generators/client/templates/vue/package.json.ejs:113]: And you should run `npm run lint-fix`

Comment (qmonmert) [reply generators/client/templates/vue/package.json.ejs:113]: I did that but there are not modifications 🧐

Comment (mshima) [reply generators/client/templates/vue/package.json.ejs:113]: npm ci?

Comment (qmonmert) [reply generators/client/templates/vue/package.json.ejs:113]: no updates 😕

Comment (mshima) [reply generators/client/templates/vue/package.json.ejs:113]: Change prettier:check script to use —debug-check.
This should print the error at actions.

Comment (qmonmert) [reply generators/client/templates/vue/package.json.ejs:113]: image

there is a warn on test/needle-api/needle-server-liquibase.spec.js but I don't see the problem

Comment (qmonmert) [reply generators/client/templates/vue/package.json.ejs]: I updated a file (package.json.ejs) and run: npm run lint-fix
=> test/needle-api/needle-server-liquibase.spec.js was updated :)

**Review by murdos — APPROVED** [review]

**Comment by DanielFran** [comment]
Thanks @qmonmert

**Pull request merged**